pkg/arvo/app/hark-store.hoon

```diff
@@ -566,10 +570,53 @@
       (~(put by last-seen) stats-index new-time)
     (give %seen-index new-time stats-index)
   ::
+  ++  get-stats-indices
+    |=  rid=resource
+    %-  ~(gas ^in *(set stats-index:store))
```

Comment:
Needing to constantly remember to do ^in is... unfortunate. I'd consider a different variable name so as not to shadow here.

Reply:
yeah ik, hopefully can get to a style refactor soonish